libs/execution_context/ebpf_native.c

```diff
@@ -833,6 +833,12 @@ _ebpf_native_reuse_map(_Inout_ ebpf_native_map_t* map)
         if (result == EBPF_KEY_NOT_FOUND) {
             ebpf_assert(handle == ebpf_handle_invalid);
             result = EBPF_SUCCESS;
+        } else {
```

Comment:
Line 856 below should call EBPF_RETURN_RESULT which will print error. Since line 835 is setting the result to success, the key_not_found case will not print any spurious error.

Reply:
Currently, in the below, there is no EBPF_RETURN_RESULT. Did you mean to add it?

```
_ebpf_native_reuse_map(..)
{ ...

Exit:
     if (result != EBPF_SUCCESS) {
        ebpf_assert_success(ebpf_handle_close(handle));
    }
    return result;
}
```